payments-core/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/TransformToPaymentMethodCreateParams.kt

```diff
@@ -20,8 +21,7 @@ internal class TransformToPaymentMethodCreateParams {
         .filterOutNullValues()
         .toMap()
         .run {
-            // TODO(brnunes): Comment why this check is needed.
-            PaymentMethodCreateParams.Type.fromCode(this["type"] as String)
+            PaymentMethod.Type.fromCode(this["type"] as String)
```

Comment:
Please still comment why this check is needed.

Reply:
What check are you referring to? It's creating a `PaymentMethod.Type` from the "type" string, because we need a `PaymentMethod.Type` for `PaymentMethodCreateParams`

Reply:
got it